lamindb/integrations/_vitessce.py

```diff
@@ -8,43 +8,60 @@
 from lamin_utils import logger
 
 from lamindb._artifact import Artifact
+from lamindb._run import Run
+from lamindb._transform import Transform
 
 if TYPE_CHECKING:
     from vitessce import VitessceConfig
 
 
 # tested & context in https://github.com/laminlabs/lamin-spatial
 def save_vitessce_config(vitessce_config: VitessceConfig, description: str) -> Artifact:
-    """Takes a ``VitessceConfig`` object and saves it as an artifact.
+    """Validates and saves a ``VitessceConfig`` object.
 
     Args:
         vitessce_config (``VitessceConfig``): A VitessceConfig object.
         description: A description for the artifact.
-    """
-    # can't assume vitessce is installed
-    from vitessce import VitessceConfig
 
-    # create a local _data export_ in a folder
-    timestamp = datetime.now(timezone.utc).isoformat().split(".")[0]
-    vitesse_export_folder = f"./vitessce_export_{timestamp}.vitessce"
-    vitessce_config.export(to="files", base_url="", out_dir=vitesse_export_folder)
-    logger.important(f"local export: {vitesse_export_folder}")
-    # create an artifact and store the local export in th cloud
-    artifact = Artifact(vitesse_export_folder, description=description)
-    artifact.save()
-    # create a JSON export that points to the data in the cloud
-    config_dict = vitessce_config.to_dict(base_url=artifact.path.to_url())
-    logger.important(f"base url: {artifact.path.to_url()}")
-    # manually place that JSON export into the local data export folder
-    config_filename = "vitessce_config.json"
-    config_file_local_path = f"{vitesse_export_folder}/{config_filename}"
+    .. versionchanged:: 0.70.2
+        This function no longer saves the dataset. It only saves the VitessceConfig object.
+    """
+    vc_dict = vitessce_config.to_dict()
+    # validate
+    datasets = vc_dict["datasets"]
```

Comment:
@ilan-gold @timtreis - if you have better ideas for validating the content of the VitessceConfig, please let me know.